File: node.js/hello.js
var http = require('http');

# Disable request logging in console output
import logging
log = logging.getLogger('werkzeug')
log.setLevel(logging.ERROR)

http.createServer(function (req, res) {
  // res.writeHead(200, {'Content-Type': 'text/plain'});
  res.end('Hello World\n');
}).listen(1337, '0.0.0.0');

console.log('Server running at http://0.0.0.0:1337/');

File: python-flask/hello.py
from flask import Flask
app = Flask(__name__)

# Disable request logging in console output
import logging
log = logging.getLogger('werkzeug')
log.setLevel(logging.ERROR)

@app.route('/')
def hello_world():
    return 'Hello World!'

if __name__ == '__main__':
    app.run(host='0.0.0.0')
